internal/dinosaur/pkg/handlers/cloud_accounts_test.go

```
@@ -31,8 +31,9 @@ func TestGetSuccess(t *testing.T) {
 		Build()
 	assert.NoError(t, err)
 	c := ocm.ClientMock{
-		GetOrganisationIDFromExternalIDFunc: func(externalID string) (string, error) {
-			return "external-id", nil
+		GetOrganisationFromExternalIDFunc: func(externalId string) (*v1.Organization, error) {
+			org, _ := v1.NewOrganization().ID("external-id").Build()
```

consideration: Can we here use `return v1.NewOrganization().ID("external-id").Build()`? I have noticed that we have a lot of cases like this in tests where we ignore errors. Maybe it's ok for us just to return what we get from `Build`.

---

I usually prefer a local variable because you can see its value during debugging.